templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List <a style="float: right" href="{% url 'taxi:manufacturer-create' %}">+</a>
```

Comment:
The '+' link for creating a new manufacturer is styled with `float: right`, which might not be visually appealing or consistent across different devices and screen sizes. Consider using Bootstrap classes like `btn` and `btn-primary` for better styling and responsiveness.